### PR TITLE
Cocktail card

### PR DESCRIPTION
**Pull request by Morel-Victor**
Create the two Component : CocktailCard and  CocktailView

### CODE REVIEW

**Review by HeloiseFerla — DISMISSED** [review]
OK but change the units to have the same units for all style.css

File: src/components/CocktailCard/style.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    height: 40vh;
+    min-height: 600px;
+    max-width: 400px;
+    display: flex;
+    flex-direction: column;
+    margin-top: 45px;
+    border-radius: 20px 20px 0 0;
+    background: #ffc700;
+
+    /* background: linear-gradient(
```
Comment (Hacene010) [inline src/components/CocktailCard/style.css]: What is this ???

File: src/components/CocktailView/index.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import CocktailCard from '../CocktailCard';
+import './style.css';
+
+export default function CocktailView() {
+  const [cocktails, setCocktails] = useState([]);
+  // const [result, setResult] = useState();
```
Comment (Hacene010) [inline src/components/CocktailView/index.jsx]: What is this ???

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Remove the commented code plz

File: src/components/CocktailCard/style.css (hunk 1 above)
Comment (PierreGoe) [reply src/components/CocktailCard/style.css]: it's remove.

File: src/components/CocktailView/index.jsx (hunk 2 above)
Comment (PierreGoe) [reply src/components/CocktailView/index.jsx]: it's remove.

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
That's better ;)

**Pull request merged**